Aliucord/src/main/java/com/aliucord/PluginManager.kt

```diff
@@ -292,7 +292,8 @@ object PluginManager {
             SupporterBadges(),
             TokenLogin(),
             UploadSize(),
-            AlignThreads()
+            AlignThreads(),
+            RemoveBilling()
```

Comment:
Could you re-alphabetize this coreplugins list? Also add a trailing comma

Reply:
They're still not alphabetized! AlignThreads is at the bottom and RemoveBilling is below RestartButton

Just get a IDE extension to sort lines and use that

Reply:
It is not
https://github.com/Aliucord/Aliucord/pull/595/files